jenkins-jobs/scripts/trigger-nightly-docker.sh

```diff
@@ -2,20 +2,28 @@
 
 DEBEZIUM_REPOSITORY=debezium/debezium
 DEBEZIUM_BRANCH=main
+DEFAULT_PLATFORMS=linux/amd64,linux/arm64
 
 SNAPSHOT_VERSION=$(curl -s https://raw.githubusercontent.com/$DEBEZIUM_REPOSITORY/$DEBEZIUM_BRANCH/pom.xml | grep -o '<version>.*-SNAPSHOT</version>' | awk -F '[<>]' '{print $3}')
 
 docker login -u ${QUAYIO_CREDENTIALS%:*} -p ${QUAYIO_CREDENTIALS#*:} quay.io
 
 # connect
-docker build --build-arg DEBEZIUM_VERSION=$SNAPSHOT_VERSION -t quay.io/debezium/connect:nightly connect/snapshot
+docker buildx build  --platform "${PLATFORM}" --build-arg DEBEZIUM_VERSION=$SNAPSHOT_VERSION -t quay.io/debezium/connect:nightly connect/snapshot
 docker push quay.io/debezium/connect:nightly
 
 # server
-docker build --build-arg DEBEZIUM_VERSION=$SNAPSHOT_VERSION -t quay.io/debezium/server:nightly server/snapshot
+docker buildx build  --platform "${PLATFORM}" --build-arg DEBEZIUM_VERSION=$SNAPSHOT_VERSION -t quay.io/debezium/server:nightly server/snapshot
```

Comment:
From where the `PLATFORM` is taken?

Reply:
It should have been `DEFAULT_PLATFORMS`. I have fixed it.